pkg/volume/cinder/attacher.go

```diff
@@ -68,9 +69,13 @@ func (attacher *cinderDiskAttacher) Attach(spec *volume.Spec, nodeName types.Nod
 
 	volumeID := volumeSource.VolumeID
 
-	instanceid, err := attacher.nodeInstanceID(nodeName)
+	instanceid, err := getNodeInstanceID(attacher.host, nodeName)
```

Comment:
It's more better if pass the full node object instead of just name.
That can save lots of requests that sends to cloud.